lib/libc/stdio/lib_sscanf.c

```diff
@@ -70,8 +70,11 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
+#define CONFIG_DYNAMIC_ALLOC_ENABLE
 
+#ifndef CONFIG_DYNAMIC_ALLOC_ENABLE
```

Comment:
As I gave my opinion to Sangam, how about use both, static 128 buffer and dynamic buffer?
Almost sscanf is used with small data, if we support only dynamic way, it causes performace down always.

If there is a way to measure the size of input, we can use dynamic way only when size is bigger than 128.